src/libs/ajax.js

```diff
@@ -92,6 +92,10 @@ const Cbas = signal => ({
       const keyParams = qs.stringify({ method_version_id: methodVersionId })
       const res = await fetchCbas(`methods?${keyParams}`, { signal, method: 'GET' })
       return await res.json()
+    },
+    post: async payload => {
```

Comment:
TOL: For future usage and maintainability, this might be a little easier to work with if we spell out the parameters and construct the payload inside the method.